transformer.py

```diff
@@ -7,16 +8,20 @@
 import os
 import random
 import time
-from typing import Optional
+from typing import Optional, Union
 import numpy as np
+from agpypeline import algorithm, entrypoint
+from agpypeline.environment import Environment
 import osr
-
 import gdal
 from osgeo import ogr
 
-import transformer_class
+from configuration import ConfigurationRgbBase
 import algorithm_rgb
```

Comment:
This produces an import error when I try to run the image. Is this supposed to be from https://github.com/AgPipeline/template-rgb-plot/blob/master/algorithm_rgb.py ? I don't remember if it was a leftover from before agpypeline was created.

Reply:
I would need a file named algorithm_rgb.py installed. This code is intended to be the foundation used by template-rgb-plot (and doesn't include the algorithm_rgb.py file)